README.rst

```diff
@@ -52,15 +52,65 @@ Quick Example
     driver.close()
 
 
+Connection Settings Breaking Change
+===================================
+
++ The driver’s default configuration for encrypted is now false (meaning that driver will only attempt plain text connections by default).
+
++ Connections to encrypted services (such as Neo4j Aura) should now explicitly be set to encrypted.
+
++ When encryption is explicitly enabled, the default trust mode is to trust the CAs that are trusted by operating system.
+
++ This means that encrypted connections to servers holding self-signed certificates will now fail on certificate verification by default.
+
++ Hostname verification is turned on by default when encryption is turned on.
+
++ The `neo4j://` scheme replaces `bolt+routing://` and can be used for both clustered and single-instance configurations with Neo4j 4.0.
+
+
+See, https://neo4j.com/docs/migration-guide/4.0/upgrade-driver/#upgrade-driver-breakingchanges
+
+
+
+Connecting with Python Driver 4.0 against Neo4j 3.5
+---------------------------------------------------
+
+Using the Python Driver 4.0 and connecting against Neo4j 3.5 with default connection settings for Neo4j 3.5.
+
+.. code-block:: python
+
+    driver = GraphDatabase.driver("neo4j+ssc://localhost:7687", auth=("neo4j", "password"))
+
+    # or
+
+    driver = GraphDatabase.driver("neo4j://localhost:7687", auth=("neo4j", "password"), encrypted=True, trust=False)
```

Comment:
Don't include this variant as a direct alternative, as the `neo4j+ssc` variant should be the preferred form. You could however label this as "is equivalent to".